code/game/objects/structures/watercloset.dm

```diff
@@ -141,6 +197,27 @@
 
 
 /obj/structure/urinal/attackby(obj/item/I, mob/user, params)
+	if(iswrench(I))
+		if(anchored)
+			user.visible_message("[user] begins disconnecting [src]...", "<span class='notice'>You begin to disconnect [src]...</span>")
```

Comment:
Pretty much nothing should be using bare black text, there are span classes that fit for pretty much every kind of message.